{"uuid":"1807ed85-1bc5-4d30-ae64-f0aa095edb99","historyId":"cdf6b598a5d7060adc71147efd8c6bcd","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"testRunner.DsalgoRunner"},{"name":"testClass","value":"testRunner.DsalgoRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testRunner.DsalgoRunner"},{"name":"host","value":"LAPTOP-2KQV9F30"},{"name":"thread","value":"30664@LAPTOP-2KQV9F30.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"BUG: Expected message 'Submission successful' but got: 'Submission Successful'","trace":"java.lang.AssertionError: BUG: Expected message 'Submission successful' but got: 'Submission Successful'\r\n\tat org.junit.Assert.fail(Assert.java:89)\r\n\tat stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:400)\r\n\tat âœ½.The user gets success message Submission successful in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"Arrays PracticeQuestion\""},{"name":"arg0","value":"\"Verify that user receives Success message on submitting valid python code for \"Max Consecutive Ones\" question\""}],"start":1746250566873,"stop":1746250571546}